manifests/g/Geph/Geph/4.8.9/Geph.Geph.installer.yaml

```diff
@@ -0,0 +1,30 @@
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: Geph.Geph
+PackageVersion: 4.8.9
+Installers:
+  - Architecture: x86
+    InstallerLocale: en-US
+    Platform:
+    - Windows.Desktop
+    MinimumOSVersion: 6.1.7601.0
+    InstallerType: inno
+    InstallerUrl: https://f001.backblazeb2.com/file/geph4-dl/geph-releases/windows-stable/4.8.9/geph-windows-setup.exe
+    InstallerSha256: B6635F47529AF2DB7EA52719E5D66F1F977093A0165493224543FB95F4D0C1FB
+    Scope: machine
+    InstallModes:
+    - interactive
+    - silent
+    - silentWithProgress
+    UpgradeBehavior: install
+    Dependencies:
+      PackageDependencies:
+      - PackageIdentifier: Microsoft.EdgeWebView2Runtime
+    ElevationRequirement: elevatesSelf
+    AppsAndFeaturesEntries:
+      - DisplayName: Geph version v4.8.9
+        DisplayVersion: 4.8.9
```

Comment:
```suggestion
```
Not recommended to use DisplayVersion when it's equal to PackageVersion. See https://github.com/microsoft/winget-pkgs/pull/65834#issue-1301374006